accio-main/src/main/java/io/accio/main/wireprotocol/WireProtocolSession.java

```diff
@@ -279,16 +280,10 @@ private void parseMetastoreSemiQuery(String statementName, String statement, Lis
     {
         String statementTrimmed = rewritePreparedChar(statement.split(";")[0].trim());
         String statementPreRewritten = PostgreSqlRewriteUtil.rewrite(statementTrimmed);
-        SessionContext sessionContext = SessionContext.builder()
-                .setCatalog(getDefaultDatabase())
-                .setSchema(getDefaultSchema())
-                .setEnableDynamic(accioConfig.getEnableDynamicFields())
-                .build();
         try {
             Statement metadataQueryStatement = MetastoreSqlRewrite.rewrite(regObjectFactory,
                     sqlParser.createStatement(statementPreRewritten, PARSE_AS_DECIMAL));
-            String converted = pgMetastore.getSqlConverter().convert(SqlFormatter.formatSql(metadataQueryStatement), sessionContext);
```

Comment:
Why don't we convert SQL here?

Reply:
In the original code base, the sql coverter from pg metastore is not with any rewriter. And remove this method is helpful for Spliting PgMetastore.